dist/jsoneditor.js

```
@@ -929,13 +929,23 @@ JSONEditor.Validator = Class.extend({
     if(typeof value === "number") {
       // `multipleOf` and `divisibleBy`
       if(schema.multipleOf || schema.divisibleBy) {
-        valid = value / (schema.multipleOf || schema.divisibleBy);
-        if(valid !== Math.floor(valid)) {
-          errors.push({
-            path: path,
-            property: schema.multipleOf? 'multipleOf' : 'divisibleBy',
-            message: this.translate('error_multipleOf', [schema.multipleOf || schema.divisibleBy])
-          });
+          var divisor = (schema.multipleOf || schema.divisibleBy);
+          var dividend = value;
+          if(parseInt(divisor, 10) < divisor) {
+              var place = (divisor + "").split(".")[1].length;
+              var multiplier = Math.pow(10, parseInt(place, 10));
+              if(dividend.toFixed(place) == dividend) {
+                  divisor = Math.round(divisor * multiplier);
+                  dividend = Math.round(dividend * multiplier);
+              }
```

If the dividend has a higher decimal placement than the divisor it can't be divided evenly. Seemed easier to ignore it and let it fail the valid check. Hard coding a 10 point based multiplier would work too and might actually be easier.